src/main/resources/org/clulab/asist/grammars/sentiments.yml

```diff
@@ -32,4 +32,13 @@ rules:
     type: token
     keep: true
     pattern: |
-      [lemma=/(?i)^(${ disagree_triggers })$/ & !tag=TO]
\ No newline at end of file
+      [lemma=/(?i)^(${ disagree_triggers })$/ & !tag=TO]
+
+
+  - name: thank_you
+    label: Gratitude
+    priority: ${ rulepriority }
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)^(${ thanking_triggers })$/ & !tag=TO]
```

Comment:
I think here you're trying to avoid things like "thanks to X" --> if so you need:
`[lemma=/thank/] (?! [tag=/TO/])`

fwiw -- I think we're overusing triggers in general, when a small list is all we need and we're unlikely to reuse it in other rules/syntactic patterns.  this is something we can consider when we re-work the grammars

Reply:
also, I think you only need `thank` as the lemma -- I am pretty sure both lemmatize to `thank`, but we should verify